doc/nrf/ug_matter_overview_network_topologies.rst

```diff
@@ -38,6 +38,12 @@ Matter network concepts
 
 The Matter network is using the following concepts, listed in alphabetical order:
 
+Binding
+  A feature that allows establishing relationship between Matter node endpoints that do not need to have matching functionalities.
```

Comment:
I think this definition doesn't convey the most important thing that the relationship is used to assign target or targets of a client cluster on the node. For example, it can be used to bind On/Off client cluster on a switch device with On/Off server cluster on a light device. Thanks to the binding, the switch device "knows" which light bulb device it should act upon.